wagtail_extensions/blocks.py

```diff
@@ -37,3 +39,52 @@ def get_context(self, value, parent_context=None):
         ctx = super().get_context(value, parent_context=parent_context)
         ctx['has_url'] = 'url' in value
         return ctx
+
+
+class AddressBlock(blocks.StructBlock):
+
+    lines = blocks.ListBlock(blocks.CharBlock(label="Line", required=False))
+
+    class Meta:
+        template = 'wagtail_extensions/blocks/address.html'
```

Comment:
Do you think it's worth specifying templates for all the blocks (without providing an actual template)?